Makefile

```diff
@@ -65,7 +65,12 @@ release-static-arm6:
 release-static-arm7:
 	mkdir -p build/release
 	cd build/release && cmake -D BUILD_TESTS=OFF -D ARCH="armv7-a" -D STATIC=ON -D BUILD_64=OFF -D CMAKE_BUILD_TYPE=release ../.. && $(MAKE)
-	
+
+release-static-armv8:
+	#May need to account for armv8.1 and armv8.2 in future
+	mkdir -p build/release
+	cd build/release && cmake -D BUILD_TESTS=OFF -D ARCH="armv8-a" -D STATIC=ON -D BUILD_64=ON -D CMAKE_BUILD_TYPE=release ../.. && $(MAKE)
```

Comment:
Yes, armv8-a is the base version and 8.1 and 8.2 add extra features on top related to half-accuracy floating point and other jazz.

It would be great if cmake could just recognise the architecture for us...and ARM don't help things by releasing half a dozen cross-compatible hardware version requiring different compiler flags to enable floating point etc.